concent_api/core/constants.py

```diff
@@ -1,6 +1,11 @@
 from enum import IntEnum
 import re
 
+# Defines available scene file paths for Concent validators
+AVAILABLE_SCENE_FILE_PATHS = ['/golem/resources/']
```

Comment:
This could have a clearer name. It's not even a full path, just a prefix.
`VALID_SCENE_FILE_PREFIXES`? `VALID_RENDERING_ROOT_DIRS`?